randovania/games/prime2/json_data/Agon Wastes.txt

```diff
@@ -2607,15 +2609,16 @@ Extra - low_memory_mode: True
       Morph Ball Bomb and Morph Ball and Space Jump Boots and Bomb Space Jump (Advanced) and Standable Terrain (Advanced)
   > Next to Pickup
       All of the following:
-          Morph Ball and Spider Ball and After Dark Samus 1 and After Main Reactor Reloaded after DS1
+          Morph Ball and After Dark Samus 1 and After Main Reactor Reloaded after DS1
           Any of the following:
-              Boost Ball
+              # https://youtu.be/q_Dwaw9d8f8
+              Morph Ball Bomb and Space Jump Boots and Bomb Space Jump (Intermediate) and Combat/Scan Dash (Advanced) and Slope Jump (Intermediate) and Standable Terrain (Advanced)
               All of the following:
-                  Morph Ball Bomb
+                  Spider Ball
                   Any of the following:
-                      Bomb Jump (Hypermode)
+                      Boost Ball
                       # https://www.youtube.com/watch?v=PHjXnyJA2is
-                      Space Jump Boots and Bomb Space Jump (Intermediate) and Standable Terrain (Intermediate)
+                      Morph Ball Bomb and Bomb Space Jump (Expert) and Standable Terrain (Intermediate)
```

Comment:
Mention this change in the changelog

Reply:
I just noticed i forgor to readd the initial trick here. Oops